tests/simpleDemo.py

```diff
@@ -45,6 +47,49 @@ def buildMITBatteryPaperPlan(datasetId):
 
     return mitPapers
 
+class VLDBPaperListing(pz.Schema):
+    """VLDBPaperListing represents a single paper from the VLDB conference"""
+    title = pz.Field(desc="The title of the paper", required=True)
+    authors = pz.Field(desc="The authors of the paper", required=True)
+    pdfLink = pz.Field(desc="The link to the PDF of the paper", required=True)
+
+def downloadVLDBPapers(vldbListingPageURLsId, outputDir):
+    """ This function downloads a bunch of VLDB papers from an online listing and saves them to disk.  It also saves a CSV file of the paper listings."""
+    policy = pz.MaxQuality()
+
+    # 1. Grab the input VLDB listing page(s) and scrape them for paper metadata
+    tfs = pz.Dataset(vldbListingPageURLsId, schema=pz.TextFile, desc="A file full of URLs of VLDB journal pages")
+    urls = tfs.convert(pz.URL, desc="The actual URLs of the VLDB pages", cardinality="oneToMany")  # oneToMany=True would be nice here.   
+    htmlContent = urls.map(pz.DownloadHTMLFunction())
```

Comment:
A minor note, which I think dovetails w/your comment in `corelib/functions.py`, is that we have now defined a `.convert()` and a `.map()` function for `pz.Dataset`. I feel like we should combine these functions into one, and if the user passes in a UDF then we execute it, and if the user does not pass in a UDF then we run the usual `.convert()` function.

Reply:
If you agree with this^ idea, a secondary issue would be to decide between keeping `.map()` or `.convert()` for a naming purpose. I have some thoughts on the pros/cons of each, but I'll wait to see if you agree first with the idea that we should merge the two.